app/controllers/scimitar/active_record_backed_resources_controller.rb

```diff
@@ -150,6 +150,16 @@ def storage_scope
         raise NotImplementedError
       end
 
+      # Return an Array of exceptions that #save! can rescue and handle with a
+      # SCIM error automatically.
+      #
+      def scimitar_rescuable_exceptions
+        [
+          ActiveRecord::RecordInvalid,
+          ActiveRecord::RecordNotSaved
```

Comment:
```suggestion
          ActiveRecord::RecordNotSaved,
          ActiveRecord::RecordNotUnique
```

Reply:
This raises the issue of exceptions that do not leave an 'offending' record with validation errors present. AFAICT, RecordNotUnique is a thin adapter-specific wrapper over the DB constraint error and does not know what attribute to mark as errant in the model, so does not add error details.

This would cause the error details in the SCIM response to be incorrect, but since it's a wider issue, I'm going to handle this more robustly and will push an update to the PR soon.